regulations/templates/regulations/comment-confirm-success.html

```diff
@@ -1,66 +1,81 @@
-Success!
-Thank you for submitting your coment to EPA. You're done!
 
-<div class="pdf">Waiting to receive PDF.</div>
+<div class="comment-confirm-success">
 
-<h3>Your comment tracking number</h3>
-<div>
-  The first step EPA takes is to assign your comment a tracking number.
-  This page automatically displays that number when received:
-</div>
-<div class="status">
-  Waiting to receive comment tracking number. This may take a few seconds
-  or minutes.
-</div>
-<div>
-  Your comment tracking number can help you find and view your public
-  comment later if you want to.
-</div>
+  <div class="success-alert">
+    <div class="fa fa-2x fa-check-circle"></div>
+    <div class="success-alert-text">
+      <h2>Success!</h2>
+      <p>Thank you for submitting your comment to EPA. You're done!</p>
+    </div>
+  </div>
 
-<h3>What will happen to your comment</h3>
-<h4>EPA will process it and in most cases will post it publicly</h4>
-<ul>
-  <li>
-    EPA will process your comment, which may take a few days or weeks depending
-    on the number of comments received. In a few cases EPA may decline to evaluate
-    or post your comment, as listed in its
-    <a href="https://www.epa.gov/dockets/commenting-epa-dockets#rules" target="_blank">
-      rules and restrictions for comments
-    </a>.
-  </li>
-  <li>
-    In most cases, EPA will post your comment to
-    <a href="#" target="_blank">the docket folder for this proposed rule</a>.
-    Then you will be able to find your comment there or by searching
-    <a href="https://www.regulations.gov" target="_blank">Regulations.gov</a>
-    for your comment tracking number or keywords from your comment.
-  </li>
-</ul>
+  <div class="save-pdf-copy">
+    <h3>Want to save a copy for yourself?</h3>
+    <a href="">Download your submitted comment (PDF)</a>
```

Comment:
The user won't be able to download the PDF upon page render; it's generated asynchronously and inserted via JS